Sources/MongoSwift/WriteConcern.swift

```diff
@@ -3,6 +3,28 @@ import Foundation
 
 /// A class to represent a MongoDB write concern.
 public struct WriteConcern: Codable {
+    /// Majority WriteConcern
+    /// - SeeAlso: https://docs.mongodb.com/manual/reference/write-concern/#writeconcern._dq_majority_dq_
+    // swiftlint:disable force_try
+    public static let majority = try! WriteConcern(w: .majority)
+    // swiftlint:enable force_try
+
+    /**
+     * Returns a customized Majority WriteConcern.
+     *
+     * - Parameters:
+     *   - wtimeoutMS: This option specifies a time limit, in milliseconds, for the write concern.
+     *   - journal: requests acknowledgment that the mongod instances, have written to the on-disk journal.
+     *
+     * - SeeAlso: https://docs.mongodb.com/manual/reference/write-concern/#writeconcern._dq_majority_dq_
+     */
+    public static func majority(wtimeoutMS: Int? = nil, journal: Bool? = nil) throws -> WriteConcern {
+        try WriteConcern(journal: journal, w: .majority, wtimeoutMS: wtimeoutMS)
+    }
+
+    /// Server default WriteConcern.
+    public static let serverDefault = WriteConcern()
+
     /// An option to request acknowledgement that the write operation has propagated to specified mongod instances.
     public enum W: Codable, Equatable {
```

Comment:
i wonder if we should be worried about forward compatibility here 🤔 

whyyyy oh whyyyy must enums be frozen

Reply:
I think in this case we're actually okay, since it would technically be a breaking change on the server side to reserve a new write concern mode due to the custom mode accepting any string.

Reply:
oh ya, good point, I forgot tag is just an arbitrary string.